src/kw_include.sh

```diff
@@ -0,0 +1,25 @@
+# Used for file sourcing,
+# accepts as argument the path for the file to be sourced
```

Comment:
Please, expand this documentation and follow the pattern used in other files (see `remote.sh`).